test/test_subprocess.py

```diff
@@ -34,7 +34,8 @@ def test_timeout():
 sys.stdout.flush()
 sys.stderr.flush()
 subprocess.call([sys.executable, "-c",
-    "import sys, subprocess; subprocess.call([sys.executable, '-c', 'import time; time.sleep(360)'])"])
+    "import sys, subprocess; subprocess.call([sys.executable, '-c', \
+    'import time; time.sleep(360)'])"])
```

Comment:
This is quite hard to read. The code itself is easy to read, we should split it in a way that makes things more clear.

I think I'd have the parameters of `call` in their own line: `sys.executable, '-c', 'import time; time.sleep(360)'`.

And I would avoid using a backslash, as this makes things harder to read, besides introducing spaces. I'd just have one string per line, which Python will automatically concatenate.